docs/source/app_developers_guide/adding_a_node.rst

```diff
@@ -0,0 +1,373 @@
+
+   .. note::
+
+      This environment uses the default location and name for this configuration
+      file. For more information, see
+      :doc:`../sysadmin_guide/configuring_sawtooth/validator_configuration_file`.
+
+#. Edit this file to change the following settings:
+
+   a. Set the ``bind`` directive to the network and component endpoints for this
+      node.
+
+      * ``network`` specifies the validator-to-validator connection (by default,
+         ``127.0.0.1:8800``
+
+      * ``component`` specifies the port on which the validator listens for
+         communication from other Sawtooth components (the REST API and
+         transaction processors)
+
+      For this application development environment, you can use the default
+      settings:
+
+      .. code-block:: none
+
+         bind = [
+           "network:tcp://127.0.0.1:8800",
+           "component:tcp://127.0.0.1:4004"
+         ]
+
+   #. Set ``endpoint`` to the IP address and port of the advertised network
+      endpoint. This is the public address that clients will use to contact this
+      validator node via the REST API.  The default value is the same as the
+      bind network value.
+
+      .. code-block:: none
+
+         endpoint = "tcp://external-ip-1:8008"
+
+   #. Comment out all other settings in this file unless your network requires a
+      non-default configuration.
+
+      For information on the validator configuration options, see "Validator
+      Configuration File" in the System Administration Guide. For more information
+      on these settings, run ``sawtooth-validator --help`` on a validator node.
+
+.. _install-second-val-ubuntu-label:
+
+Step 2: Install and Configure a Second Validator Node
+------------------------------------------------------
+
+#.  Install and configure Sawtooth on the second node as described in
+    :doc:`ubuntu`, with one difference: **Do not** create a genesis block.
+
+#. Open a new terminal window on the second node and start the Validator
+   Registry transaction processor with this command:
+
+   .. code-block:: console
+
+      user@validator-registry$ sudo -u sawtooth sawtooth-poet-validator-registry-tp -v
+      [23:47:06 INFO    core] register attempt: OK
+
+
+#. Create the ``validator.toml`` configuration file on the second validator
+   node. You can copy this file from the first validator node or start with
+   the example configuration file, as in the following command:
+
+   .. code-block:: console
+
+      user@valnode2$ sudo cp /sawtooth-core/packaging/validator.toml.example /etc/sawtooth/validator.toml
+
+#. Edit this file to change the following settings:
+
+   a. Set ``bind`` to the network and component endpoints for this node. For
+      this application development environment, can use the default values.
+
+      .. code-block:: none
+
+         bind = [
+           "network:tcp://127.0.0.1:8008"
```

Comment:
8800 is default in our example toml file. Someone pipe up if we have a different default specified elsewhere.